presto-main/src/main/java/io/prestosql/sql/analyzer/StatementAnalyzer.java

```diff
@@ -682,37 +683,43 @@ protected Scope visitResetSession(ResetSession node, Optional<Scope> scope)
         @Override
         protected Scope visitAddColumn(AddColumn node, Optional<Scope> scope)
         {
+            analysis.setUpdateType("ADD COLUMN", createQualifiedObjectName(session, node, node.getName()));
```

Comment:
"ADD COLUMN" update type is also configured from `io.prestosql.execution.DataDefinitionTask#getName` also